### PR TITLE
Better Documentation: Period Endings

### PR DESCRIPTION
**Pull request by ahmadawais**
As per the [PHP documentation standards in the core handbook](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/), this commit helps make the documentation better by adding:

- Period endings
- A Long description

### CODE REVIEW

File: includes/widgets.php [hunk 1]
```diff
@@ -113,7 +115,7 @@ function __construct() {
 
 	/** @see WP_Widget::widget */
 	function widget( $args, $instance ) {
-		// Set defaults
+		// Set defaults,
```
Comment (pippinsplugins) [inline includes/widgets.php]: This should not be a comma

**Comment by ahmadawais** [comment]
@pippinsplugins My bad. Just updated it.

**Comment by ahmadawais** [comment]
@pippinsplugins Can you look into the updated PR.

**Comment by pippinsplugins** [comment]
@ahmadawais looks good.

**Comment by ahmadawais** [comment]
:)

**Pull request merged**